_posts/2019-08-29-cncf.md

```diff
@@ -24,7 +24,11 @@ We hope that membership of the CNCF will allow many more users to adopt Apache K
 
 The Strimzi community is friendly and driven by the desire to further Open Source principles - we are always open to new contributions.
 Please do get involved, not just through code; documentation improvements, bug reports and blog posts are all appreciated.
-You can download the latest version of Strimzi [here](https://strimzi.io/downloads/) and say hello on [Slack](https://join.slack.com/t/strimzi/shared_invite/enQtMzU2Mjk3NTgxMzE5LTYyMTUwMGNlMDQwMzBhOGI4YmY4MjhiMDgyNjA5OTk2MTFiYjc4M2Q3NGU1YTFjOWRiMzM2NGMwNDUwMjBlNDY).
+
+As Strimzi moves to the CNCF we are taking the opportunity to make a few changes to the communication methods we use. 
+Firstly, we will be moving to a [CNCF hosted Slack channel](https://cloud-native.slack.com/messages/CMH3Q3SNP) which, as an enterprise version, has unlimited users and message history. 
+We understand that moving the Slack channel is painful for all the users of the current channel, but feel that in this case, the benefits outweigh the pain. 
+Secondly, we will be moving to a new mailing list, again hosted by CNCF. The new mailing list will be [strimzi-dev@lists.cncf.io](mailto://strimzi-dev@lists.cncf.io). 
```

Comment:
it should be `cncf-strimzi-dev@lists.cncf.io` (so with the `cncf-` prefix) but I asked to remove the `-dev` suffix, let's see ...

Reply:
Updated to be the correct address but will change again if the `-dev` is removed